docs_requirements.txt

```diff
@@ -7,3 +7,5 @@ sphinx-rtd-theme
 nbsphinx!=0.8.8
 nbsphinx_link
 ipython
+# temporary for https://github.com/jupyter/nbconvert/issues/2092
+nbconvert<7.14
```

Comment:
Seems like it might be fixed ? I've unpinned `nbconvert` and it tests fine.

I think we already switched to `tqdm.auto` upstream in LiberTEM, but these notebooks have not bee regenerated since.